ape_alchemy/provider.py

```diff
@@ -129,22 +136,37 @@ def get_virtual_machine_error(self, exception: Exception) -> VirtualMachineError
         return VirtualMachineError(message=message)
 
     def _make_request(self, endpoint: str, parameters: list) -> Any:
-        try:
-            return super()._make_request(endpoint, parameters)
-        except HTTPError as err:
-            response_data = err.response.json()
-            if "error" not in response_data:
-                raise AlchemyProviderError(str(err)) from err
-
-            error_data = response_data["error"]
-            message = (
-                error_data.get("message", str(error_data))
-                if isinstance(error_data, dict)
-                else error_data
-            )
-            cls = (
-                AlchemyFeatureNotAvailable
-                if "is not available" in message
-                else AlchemyProviderError
-            )
-            raise cls(message) from err
+        MIN_RETRY_DELAY = 1000
```

Comment:
In Python, constants are typically capitalized only when defined at the global scope.
It may make more sense to make all of these function arguments on `_make_request()` so that hackers can change their values if they want to get all internal on us.

Reply:
converted to lower case and put as default callable parameters